electron/utils.ts

```diff
@@ -549,11 +549,12 @@ const formatEpicStoreUrl = (title: string) => {
 }
 
 function quoteIfNecessary(stringToQuote: string) {
-  if (
-    stringToQuote &&
+  const shouldQuote =
+    typeof stringToQuote === 'string' &&
```

Comment:
added one more check here because the function was breaking because sometimes it was receiving an object or array as a variable.